__tests__/server.test.js

```diff
@@ -518,6 +518,72 @@ describe("/api/comments/:comment_id", () => {
             })
         })
     })
+    describe("PATCH", () => {
+        test("200: Increments the vote count by the given amount", () => {
+            return request(app)
+            .patch("/api/comments/1")
+            .send({inc_votes: 4})
+            .expect(200)
+            .then((response) => {
+                expect(response.body.updatedComment.votes).toBe(20)
+            })
```

Comment:
You could try to destructure perhaps? i.e.

.then(({ body}) => { body.updatedComment.votes}

or a step further

.then({body: {updatedComment}}) => { updatedComment.votes}

etc